src/Components/App/App.js

```diff
@@ -0,0 +1,23 @@
+import React, {useState} from 'react'
+import { Routes, Route } from 'react-router-dom'
+import Search from '../Search/Search'
+import Results from '../Results/Results'
+import Album from '../Album/Album'
+import './App.css';
+
+function App() {
+  const [query, setQuery] = useState('')
+  
+  return (
+    <div className="App">
+      <Search setQuery={setQuery}/>
```

Comment:
I've gotten comments saying this is not best practice, It is generally considered better practice to handle state management within the component that owns the state, rather than passing setter functions from outside the component.

Reply:
nice catch, upon thinking about this more I got rid of the query state and used useParams instead, now the url reflects the query and the page, this fixed the refresh bug